vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/payment_util_unittest.cc

```diff
@@ -3,9 +3,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "base/command_line.h"
 #include "bat/ledger/global_constants.h"
 #include "bat/ledger/internal/endpoint/payment/payment_util.h"
 #include "bat/ledger/ledger.h"
+#include "brave/components/brave_rewards/browser/switches.h"
```

Comment:
I think you can avoid this by not checking the switch directly in ledger as mentioned above

Reply:
resolved: https://github.com/brave/brave-core/pull/7557/files#diff-5f0e465a3cf0ddbe66878d7371c404e46db82c8612b7551e9763a12aa5d6d84d